stdlib/turtle.pyi

```diff
@@ -18,7 +18,17 @@ _PolygonCoords = Sequence[tuple[float, float]]
 # Vec2D is actually a custom subclass of 'tuple'.
 Vec2D = tuple[float, float]
 
-class ScrolledCanvas(Frame): ...
+# Does not actually inherit from Canvas, but dynamically gets all methods of Canvas
+class ScrolledCanvas(Canvas):
```

Comment:
This class represents a canvas, but with scrollbars on two sides. There is a tkinter trick that makes it possible to do that and actually inherit from `Canvas`, with much less method aliasing than turtle does, but it's far from obvious and whoever wrote turtle probably didn't know it.

Frame is a very minimal widget in Tk. It does only a couple things that canvas doesn't do, so not inheriting from `Frame` shouldn't leave out anything important.

Reply:
If you're lying in the stub anyway, could you use multiple inheritance to inherit from both?

Reply:
It now has multiple inheritance that just looks wrong to me, knowing how tkinter widgets work, but maybe it servers the typing better :)